framework/src/bundle/BundlePrivate.cpp

```diff
@@ -507,6 +507,9 @@ std::exception_ptr BundlePrivate::Start0()
       res = std::make_exception_ptr(cppmicroservices::SharedLibraryException(
         ex.code(), ex.what(), thisBundle));
     } catch (...) {
+      coreCtx->logger->Log(logservice::SeverityLevel::LOG_INFO,
```

Comment:
I think we want to use the Log overload that takes an exception_ptr so that the exception is also logged.